src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -251,6 +253,12 @@ describe('non-empty successful affected clusters table', () => {
         });
     });
   });
+
+  it('sorts N/A in last seen correctly', () => {
+    cy.get('.pf-c-table__sort').eq(1);
```

Comment:
What's the point of this line? Don't you need to perform a click or?..

Reply:
Yep, this was a part of the test I deleted to check how the sorting works when I discovered a bug